src/leveled_pmem.erl

```diff
@@ -201,16 +201,16 @@ merge_trees(StartKey, EndKey, TreeList, LevelMinus1) ->
 
 find_pos(<<>>, _Hash) ->
     false;
-find_pos(<<1:1/integer, Hash:23/integer, _T/binary>>, Hash) ->
+find_pos(<<Hash:24/integer, _T/binary>>, Hash) ->
```

Comment:
This can now return `true` if the hash's most significant bit it `0`. In the original code, it would then fail. But in the original code, the bit is set to 1 in case of prepare_for_index... a bit hard to grasp what is going on here

Reply:
The prepending of "1" had no value ... probably a legacy of the related to the use of find_pos in `leveled_sst`.